src/ui/common/src/components/pages/artifact/id/index.tsx

```diff
@@ -111,7 +111,9 @@ const ArtifactDetailsPage: React.FC<ArtifactDetailsPageProps> = ({
 
   useEffect(() => {
     if (!!artifact) {
```

Comment:
Should this condition be `if (!!artifact && !sideSheetMode)` like what we do for the operator detail pages?

Reply:
yes, good catch, i'll update this.